lua/noice/config/init.lua

```diff
@@ -262,6 +262,9 @@ function M.setup(options)
     end,
   })
 
+  if M.options.cmdline.enabled then
+    vim.o.cmdheight = 0
```

Comment:
Why is this needed?

Reply:
If I don't apply it here, I guess due to the fact that MiniView's initialisation happens too early, there's a gap between the pop-ups and `cmdline`.

<img width="261" alt="image" src="https://github.com/folke/noice.nvim/assets/4507647/7cee8ced-e985-4824-be27-abfefb057131">

PS: repro.lua
```lua
-- DO NOT change the paths and don't remove the colorscheme
local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({ "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", lazypath, })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
  "folke/tokyonight.nvim",
  "MunifTanjim/nui.nvim",
  { "https://github.com/Frederick888/noice.nvim.git", branch = 'mini-dodge-status-line' },
  -- add any other plugins here
}
require("lazy").setup(plugins, {
  root = root .. "/plugins",
})

vim.cmd.colorscheme("tokyonight")
-- add anything else here

require("noice").setup({
  lsp = {
    -- override markdown rendering so that **cmp** and other plugins use **Treesitter**
    override = {
      ["vim.lsp.util.convert_input_to_markdown_lines"] = true,
      ["vim.lsp.util.stylize_markdown"] = true,
      ["cmp.entry.get_documentation"] = true, -- requires hrsh7th/nvim-cmp
    },
  },
  -- you can enable a preset for easier configuration
  presets = {
    bottom_search = true, -- use a classic bottom cmdline for search
    command_palette = true, -- position the cmdline and popupmenu together
    long_message_to_split = true, -- long messages will be sent to a split
    inc_rename = false, -- enables an input dialog for inc-rename.nvim
    lsp_doc_border = false, -- add a border to hover docs and signature help
  },

  cmdline = {
    enabled = true,
  },
  messages = {
    enabled = true,
  }
})

local timer = vim.uv.new_timer()
local counter = 1
vim.uv.timer_start(timer, 1000, 1000, function ()
  vim.notify('hello, world! ' .. counter)
  counter = counter + 1
end)
```